.circleci/config.yml

```diff
@@ -154,7 +154,7 @@ jobs:
           command: npm version ${CIRCLE_TAG} --workspace=packages/
       - run:
           name: NPM publish
-          command: npm publish --access=public --tag=pre-release
+          command: npm publish --workspace=packages/ --access=public --tag=prerelease
```

Comment:
**question**: why are we using a different tag now? the previous script [used `pre-release`](https://github.com/Financial-Times/dotcom-page-kit/blob/v7.2.7/scripts/get-npm-dist-tag.js#L35)

Reply:
It did! I will undo that.

Reply:
Fixed.